src/cluster_legacy.h

```diff
@@ -303,6 +315,8 @@ struct _clusterNode {
     mstime_t orphaned_time;                 /* Starting time of orphaned primary condition */
     long long repl_offset;                  /* Last known repl offset for this node. */
     char ip[NET_IP_STR_LEN];                /* Latest known IP address of this node */
+    sds client_ipv4;                        /* IPv4 for clients only. */
+    sds client_ipv6;                        /* IPv6 for clients only. */
```

Comment:
The client_ip* names sound confusing to me. can we add `announce` to make them more explicit?
```suggestion
    sds client_announce_ipv4;                        /* IPv4 for clients only. */
    sds client_announce_ipv6;                        /* IPv6 for clients only. */
```

Reply:
`announce_client_ipv4` to match the config name